parsely/src/main/java/com/parsely/parselyandroid/ParselyAPIConnection.kt

```diff
@@ -0,0 +1,62 @@
+/*
+    Copyright 2016 Parse.ly, Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+package com.parsely.parselyandroid
+
+import android.os.AsyncTask
+import java.net.HttpURLConnection
+import java.net.URL
+
+@Suppress("DEPRECATION")
+internal class ParselyAPIConnection(private val tracker: ParselyTracker) : AsyncTask<String?, Exception?, Void?>() {
+    private var exception: Exception? = null
+
+    @Deprecated("Deprecated in Java")
```

Comment:
Question (❓): Why did you chose `@Deprecated(...)`, which has a different usage, instead of just using `@Suppress("OVERRIDE_DEPRECATION")`? 🤔

PS: Same applies to `onPostExecute(...)` function further below.